tw-curator-starter/src/test/java/com/transferwise/common/curator/CuratorConfigurationPropertyMissingTest.java

```diff
@@ -0,0 +1,21 @@
+package com.transferwise.common.curator;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(classes = {TestApplication.class})
+public class CuratorConfigurationPropertyMissingTest {
```

Comment:
Lets change it to *IntTest

Reply:
I notice, that other integration tests are also just named *Test. But our convention in other services and libraries has been to use IntTest for integration tests.